scripts/vscripts/tf2ware_ultimate/minigames/avoid_barrels.nut

```diff
@@ -0,0 +1,65 @@
+minigame <- Ware_MinigameData
+({
+	name          = "Explosive Barrels"
+	author		  = ["sasch", "ficool2", "tilderain"]
```

Comment:
sasch shouldn't be credited here as it's a brand new minigame (even if its similar concept, code etc), unless this is taken from a TF2Ware version I'm not aware of